README.md

```diff
@@ -483,6 +483,17 @@ the following metadata tag to your manifest file.
 </manifest>
 ```
 
+#### Proguard / R8 Issues
+
+If you notice issues in the release build of your apps, you can try to manually add a couple rules
+to your `proguard-rules.pro` to prevent obfuscation:
+```
+-keep class com.klaviyo.analytics.** { *; }
+-keep class com.klaviyo.core.** { *; }
```

Comment:
and you need to also add this to core?

Reply:
Since core has no dependencies (not including tests) I think we are okay here (since there's nothing it could accidentally obfuscate)

Reply:
I'm like 99% that they shouldn't need `-keep class com.klaviyo.core.** { *; }` because they shouldn't have any reference to core in their code. I guess if they're troubleshooting, maybe better to be safe?

Reply:
Yeah I think just because we do expose it this wouldn't be the worst idea, but certainly can remove it if you'd like

Reply:
Having re-read the article, I feel pretty confident no one should ever need any of these three rules based if they are consuming the Android SDK directly.
BUT this is the troubleshooting section -- so its good to cover our bases. So I say keep all three like you have it!